cmd/validate-yaml/main_test.go

```diff
@@ -0,0 +1,139 @@
+package validate_yaml
+
+import (
+	"bytes"
+	"errors"
+	"testing"
+
+	pkgExtractor "github.com/datreeio/datree/pkg/extractor"
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
+
+type MockFileReader struct {
+	mock.Mock
+}
+
+func (fr *MockFileReader) FilterFiles(paths []string) ([]string, error) {
+	fr.Called(paths)
+	return paths, nil
+}
+
+type MockPrinter struct {
+	mock.Mock
+}
+
+func (p *MockPrinter) PrintFilename(title string) {
+	p.Called(title)
+}
+
+func (p *MockPrinter) PrintYamlValidationErrors(validationErrors []error) {
+	p.Called(validationErrors)
+}
+
+func (p *MockPrinter) PrintYamlValidationSummary(passedFiles int, allFiles int) {
+	p.Called(passedFiles, allFiles)
+}
+
+func (p *MockPrinter) PrintMessage(messageText string, messageColor string) {
+	p.Called(messageText, messageColor)
+}
+
+type MockExtractor struct {
+	mock.Mock
+}
+
+func (e *MockExtractor) ExtractConfigurationsFromYamlFile(path string) (*[]pkgExtractor.Configuration, string, *pkgExtractor.InvalidFile) {
+	args := e.Called(path)
+	var firstArgument *[]pkgExtractor.Configuration
```

Comment:
I think we can use something more informative as variable name :)

Reply:
changed!